Automattic/wp-calypso
# Marketplace: Improve partner submission success page

#### Proposed Changes

We need to iterate on https://wordpress.com/marketplace/submission-success to match the new design.
Figma: SJ6I0rUUtHNdHMF2KMMJig-fi-456%3A10526

#### Testing Instructions

1. Go to `/marketplace/submission-success`
2. Check the design to match figma
3. Check `Developer Page` should redirect to `developer.wordpress.com`
4. Check the small footer and links should match links in our [existing partner’s page form.](https://wordpress.com/partners/)

#### Screenshots
##### Desktop
![image](https://user-images.githubusercontent.com/402286/172686637-c1b13bca-cfeb-42ab-ba95-df5d1708b658.png)

##### Mobile
<img width="196" alt="image" src="https://user-images.githubusercontent.com/402286/172687025-7b4e5d3a-b494-4b5c-bbc4-dc67e9d5c98e.png">



Fixes #64033
